### DISCUSSION TITLE
Restore YouTube playback with screen off?

### DISCUSSION THREAD

**Discussion by kilpatrq** [ReVanced/revanced-discussions#1074] [Question] [answered]
I last patched YouTube about three months ago and decided it was time for an update today. Saw that the apk i used last time was still the recommended apk, so no reason to change it. However, after updating Revanced Manager and patching, and importing all my old settings, I can no longer turn my screen off and still get playback. This feature worked fine this morning, on my three month old build. How do I restore it? I used the exact same patches both times, though i did note that video spoofing had been removed from my patch list and had to be manually selected this time.

**Comment by oSumAtrIX** [comment, marked as answer]
Enable the background playback patch
